Modules/Core/Common/include/itkBooleanStdVector.h

```diff
@@ -19,6 +19,7 @@
 #define itkBooleanStdVector_h
 
 #include "itkIntTypes.h"
```

Comment:
`#include "itkIntTypes.h"` was only there for `uint8_t`, so if you want to replace it by `ThreadSafeBooleanType`, you might as well remove `#include "itkIntTypes.h"` !

Reply:
Good catch.  Thank you.